code/drasil-website/Drasil/Website/Example.hs

```diff
@@ -85,17 +105,16 @@ exampleDoxRefs path =[[(pendulum, [])],
                  [(template, [])]]
 
 --example names, maybe make a unique type to accept fields of documents, gen code, and doxygen?
-pendulum, gamePhys, glassBR, hghc, noPCM, pdController, projectile, projectileC1,
-  projectileC2, projectileC3, projectileC4, projectileC5, ssp, swhs, template :: String
-
+pendulum, gamePhys, glassBR, hghc, noPCM, pdController, projectile, ssp, swhs, template :: String
+projectileC1, projectileC2, projectileC3, projectileC4, projectileC5 :: String
 pendulum = "DblPendulum"
 gamePhys = "GamePhysics"
 glassBR = "GlassBR"
 hghc = "HGHC"
 noPCM = "NoPCM"
 pdController = "PDController"
 projectile = "Projectile"
-projectileC1 = "Projectile_C_P_NoL_B_U_V_D"
+projectileC1 = "Projectile_C_P_NoLB_U_V_D"
```

Comment:
Are we able to pull these strings from the examples SystemInformation/Choices? If not, we might want to figure out what information that the SIs are missing (si name, si description, choice names?) and add them (or tie everything together in something else that the drasil-website can better digest) so that we can pull them.

Reply:
Everything is looking nice! Looking forward to seeing what's next for the website!

Reply:
Oh, yes those strings should eventually move out - I've only really looked at the `CaseStudy.hs` file so far, but `Example.hs` will definitely be changed.